### PR TITLE
Documentation for the rotating-pulsating frame

### PR DESCRIPTION
**Pull request by eggrobin**
#3358.

### CODE REVIEW

File: documentation/basestyle.cls [hunk 1]
```diff
@@ -22,7 +22,7 @@
 % Language.
 \RequirePackage{polyglossia}
 \setdefaultlanguage[variant=british]{english}
-\setotherlanguages{greek,german,russian,latin,french}
```
Comment (pleroy) [inline documentation/basestyle.cls:25]: Quousque tandem?

File: documentation/bibliography.bib [hunk 2]
```diff
@@ -1710,6 +1721,15 @@ @book{Sterbenz1974
   title     = {Floating-point computation},
 }
 
+@book{Szebehely1967,
+  author    = {Szebehely, Victor G.},
+  publisher = {Academic Press},
+  date      = {1967},
+  isbn      = {978-0-12-395732-0},
+  subtitle  = {The restricted problem of three Bodies},
```
Comment (pleroy) [inline documentation/bibliography.bib:1729]: The Restricted Problem of Three Bodies, see https://www.sciencedirect.com/science/article/pii/B9780123957320500015.

File: documentation/Rotating Pulsating.tex [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+\deriv[2]{ν}\vq^{\mathscr P}
+=\derivop{ν}\pa{\deriv{t}\vq^{\mathscr P}\deriv{ν}{t}}
+&=\deriv[2]{t}\vq^{\mathscr P}\pa{\deriv{ν}{t}}^2
+ +\deriv{t}\vq^{\mathscr P}\deriv[2]{ν}{t}\\
+&=\deriv[2]{t}\vq^{\mathscr P}\pa{\deriv{ν}{t}}^2
+ +\deriv{t}\vq^{\mathscr P}\pa{\derivop{t}\deriv{ν}{t}}\deriv{ν}{t}\\
+&=\frac{\SecondTimeDerivative\vq^{\mathscr P}}{\TimeDerivative ν^2}
+ +\TimeDerivative\vq^{\mathscr P}\pa{\derivop{t}\frac{1}{\TimeDerivative ν}}\frac{1}{\TimeDerivative ν}\\
+&=\frac{\SecondTimeDerivative\vq^{\mathscr P}}{\TimeDerivative ν^2}
+ \TimeDerivative\vq^{\mathscr P}\frac{\SecondTimeDerivative ν}{\TimeDerivative ν^2}\frac{1}{\TimeDerivative ν}\\&=\frac{1}{\TimeDerivative ν^2}\pa{\SecondTimeDerivative\vq^{\mathscr P}
```
Comment (pleroy) [inline documentation/Rotating Pulsating.tex:325]: Missing minus sign at the beginning of this line.

File: documentation/basestyle.cls (hunk 1 above)
Comment (eggrobin) [reply documentation/basestyle.cls:25]: For reasons that I do not want to investigate, Latin breaks _something_ we do:
```
Rotating Pulsating.tex
26
Undefined control sequence.
^->\active@prefix ^\@undefined 
                               
l.26 ...raph*{Derivative} The notation $\deriv x f
                                                  $ represents the evaluation
```

At the same time, we appear not to be using Latin, even where we have actual Latin, such as in cbrt.tex:
https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L222-L242 https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L1270-L1275 https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L1436-L1442

Contrast French, where we use `\textfrench`:
https://github.com/mockingbirdnest/Principia/blob/cee0532d896e0cdd45701cc0328f1237010f5793/documentation/cbrt.tex#L337-L340

**Pull request merged**